engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/cache/DefaultSchemaCache.scala

```diff
@@ -0,0 +1,34 @@
+package pl.touk.nussknacker.engine.avro.cache
+
+import com.github.benmanes.caffeine.cache.{Cache, Caffeine}
+import org.apache.avro.Schema
+import scalacache.Entry
+
+import scala.concurrent.duration.Duration
+
+class DefaultSchemaCache extends SchemaCache {
+
+  import scalacache.caffeine._
+  import scalacache.modes.sync._
+
+  private val caffeineClientCache: Cache[String, Entry[Schema]] =
+    Caffeine
+      .newBuilder()
+      .maximumSize(10000L)
```

Comment:
Can you extract it in constructor and set keep it in const in `CachedConfluentSchemaRegistryClient`?

Reply:
Good Idea!